benchmark/sparse/linear_regression/keras_sparse_model.py

```diff
@@ -23,7 +25,8 @@ def run_benchmark(train_data, train_label, eval_data, eval_label, batch_size, ep
               epochs=epochs,
               batch_size=batch_size)
 
-    print("Keras-TF Sparse Benchmark Results")
+    print("Keras Sparse Benchmark Results")
+    print("Backend: ", K.backend())
     print("Batch Size")
```

Comment:
Keep logging style consistent for backend, batch size, etc. print("Name: ", value)

Reply:
fixed